modules/search/instant-search/components/search-filters-widget.jsx

```diff
@@ -60,8 +60,10 @@ export default class SearchFiltersWidget extends Component {
 
 	render() {
 		const aggregations = get( this.props.results, 'aggregations' );
+		const style = this.props.loading === true ? { opacity: 0.2 } : { opacity: 1 };
```

Comment:
Shall we apply this via a class name rather than applying the style inline? `is-loading`?